apps/GNOME/credits

```diff
@@ -0,0 +1 @@
+TerraGitHuB (inital script) - YT channel: https://www.youtube.com/channel/UCjCHOhXCgQRBHCY8IeBZTzQ
```

Comment:
no credit shoud be given to anyone but the gnome developers
this isn't special or unique

Reply:
I was wondering what the credits should be. I didn't add/remove parties in the credits as I wasn't sure.